### DISCUSSION TITLE
Join .NET Foundation

### DISCUSSION THREAD

**Discussion by kayoub5** [chmorgan/sharppcap#340] [Ideas]
I think project mostly qualify to be a .NET Foundation project, should we go for it?

# Benefits
* Dedicated Azure DevOps instance, which allows projects to use as many agents in Pipelines as they need.
https://dotnetfoundation.org/projects/why-join
* Sponsored a number of MSDN subscriptions for .NET Foundation project contributors
* Code signing
* Help with IP and legal
* Marketing and communications 
* More at https://dotnetfoundation.org/projects/why-join

# Eligibility
- [ ] Source Link
- [ ] Symbol packages 
- [ ] Code Signing
- [ ] Contributor License Agreement
- More at https://dotnetfoundation.org/projects/net-foundation-project-committee

**Comment by chmorgan** [comment]
I'm not opposed to joining. @PhyxionNL, thoughts?

**Reply by PhyxionNL** [reply]
Sounds like a good idea, I think there are a couple of points left to fully qualifiy.

**Comment by kayoub5** [comment]
Source Link and symbol packages should be resolved with #346, pending tests

@chmorgan for CLA see:
* https://github.com/cla-assistant/cla-assistant
* http://harmonyagreements.org/

**Comment by chmorgan** [comment]
I'm thinking we might want to avoid the .NET foundation. They had that PR merged story but this one is way more of a concern, https://robmensching.com/blog/posts/2021/10/6/how-the-.net-foundation-kerfuffle-became-a-brouhaha/

Thoughts?

**Reply by PhyxionNL** [reply]
I don't think transfering ownership to the foundation is a good idea. Until that changes, let's avoid it?

**Reply by kayoub5** [reply]
Agreed.
